level2/restaurants/kubo_restaurants.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Restaurant class
+class Restaurant
+  attr_reader :name, :items
+
+  def initialize(name)
+    @name = name
+    @items = []
+  end
+
+  def add_items(*items)
+    items.each { |item| @items << item }
+  end
+
+  def remove_item(item_name)
+    @items.reject! { |item| item.name == item_name }
+  end
+
+  def place_order(order_items, order_discount = 0)
+    order_discount = order_discount[:discount] if order_discount != 0
```

Comment:
this is an interesting solution, but there is a simpler way to do this with [keyword arguments](https://github.com/pmkaboo/coding-classes/blob/main/level2/self_study/keyword_arguments.rb)